presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7869,6 +7873,29 @@ public void testQuantifiedComparison()
         assertQuery("SELECT cast(1 as decimal(3,2)) <> ANY(SELECT cast(1 as decimal(3,1)))");
     }
 
+    @Test(dataProvider = "quantified_comparisons_corner_cases")
+    public void testQuantifiedComparisonCornerCases(String query)
+    {
+        assertQuery(query);
+    }
+
+    @DataProvider(name = "quantified_comparisons_corner_cases")
+    public Object[][] qualifiedComparisonsCornerCases()
+    {
+        List<QueryTemplate.Parameter> subquery = new QueryTemplate.Parameter("subquery").of(
+                "SELECT * FROM (SELECT 1 WHERE false) as empty_table",
```

Comment:
I wanted to use the alias for explaining what case is tested but I think it's not a that much needed in this case